plugins/module_utils/net_map/networking_mapper.py

```diff
@@ -610,7 +627,7 @@ def __build_network_tools(
             else None
         )
         multus = (
-            self.__build_network_tool_common(
+            self.__build_network_tool_route(
```

Comment:
the netconfig tool also supports configuring routes so I would like to test switching it to `__build_network_tool_route` as well